pymaker/keys.py

```diff
@@ -0,0 +1,66 @@
+# This file is part of Maker Keeper Framework.
+#
+# Copyright (C) 2018 reverendus
+#
+# This program is free software: you can redistribute it and/or modify
+# it under the terms of the GNU Affero General Public License as published by
+# the Free Software Foundation, either version 3 of the License, or
+# (at your option) any later version.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU Affero General Public License for more details.
+#
+# You should have received a copy of the GNU Affero General Public License
+# along with this program.  If not, see <http://www.gnu.org/licenses/>.
+
+import getpass
+from typing import Optional
+
+from eth_account import Account
+from web3 import Web3
+from web3.middleware import construct_sign_and_send_raw_middleware
+
+from pymaker import Address
+
+_registered_accounts = {}
+
+
+def register_keys(web3: Web3, keys: Optional[list]):
+    def not_none(x):
+        return x if x is not None else []
+
+    for key in not_none(keys):
```

Comment:
You could make it like that
```suggestion
    for key in keys or []:
```